### ISSUE TITLE
microbiome sampling sat. 4/8?

### ISSUE THREAD

**Issue by yaaminiv** [sr320/LabDocs#565]
@laurahspencer and I think it may be a good idea to get microbiome samples this Saturday, especially because we've seen some weird stringy stuff in two tanks (see below):

![stringy stuff](https://cloud.githubusercontent.com/assets/22335838/24777008/59bbb842-1ad8-11e7-9d67-0e3334128ed2.JPG)

I'm looking at this protocol: https://docs.google.com/document/d/1v9c7yEtXZ-eKGE6zFOkY7U1R7dc5mGe5Ru8nCp5INh0/edit

From what I can understand, we'll need filter paper, GFF with 0.5 pore size to remove larger particulates, followed by a 0.2, in terms of equipment. @sr320 or @kubu4 do we have any of that available? Or is this not possible on such short notice?

**Comment by sr320** [comment]
@Ellior2 is a pro at this, and would be able to tell you what supplies you would need.

One key consideration is access to perstalic pump or vacuum to move water

**Comment by sr320** [comment]
You could also preserve stringy stuff to try to ID

**Comment by yaaminiv** [comment]
@laurahspencer do you know if we'd have access to such a pump at manchester?

@sr320 so just stick it in a tube, flash freeze it?

**Comment by sr320** [comment]
Yes, and maybe a second tube in EtOH

**Comment by yaaminiv** [comment]
from @laurahspencer: is this a pump we can use?

![51321596342__2beac709-44c9-4fb7-9743-0b76c010ef3a jpg](https://cloud.githubusercontent.com/assets/22335838/24781731/78fc7d36-1af6-11e7-8e23-c7061687cefa.jpeg)

She thinks it's a vacuum pump: http://gastmfg.com/uploads/catalogs/F-11_DiaphragmVacuum_Pump_Catalog_Final_Feb09-2012_lores.pdf

**Comment by sr320** [comment]
Located?

**Comment by yaaminiv** [comment]
@sr320 at PSRF

**Comment by sr320** [comment]
Would have to ask PSRF

**Comment by Ellior2** [comment]
Microbiome sampling (the way I did it) is quite involved. I guess if you were just doing one or two samples of the header it might not be so bad. Peristaltic pumps work much better. I have made a filter apparatus but we are kind of short on pumps at the moment, so I wouldn't be able to bring one. Brent grabbed the filters today from the lab I think to bring to Taylor next week so he's probably got them at his house or something. Are you guys cleaning the headers and vacuuming broodstock regularly? Has anyone else at PSRF seen this stringy stuff? Is it everywhere throughout the hatchery or are you just seeing it in your header? At the very least just grab a sample of it and look at it under the microscope. I'll be there Saturday, sorry I haven't been all that involved with your projects out there. I am excited to see it though!

**Comment by Ellior2** [comment]
Just remembered I've actually got some leftover filters and could bring supplies out if you guys want. @sr320  since Brook said to not use TCA, I don't know what the new method of preservation is- just putting them in the -80? If you guys can get a peristaltic pump (with the correct sized masterflex tubing) and ethanol for sanitization, I can get the rest.

**Comment by sr320** [comment]
Yes just -80
There is  currently a peristaltic pump being used for feeding- presume is
could be sanitized.

**Comment by Ellior2** [comment]
Another question I have is are you only seeing it in one of your treatments?

**Comment by laurahspencer** [comment]
We are seeing the stringy stuff only in the low pH tanks over the past few weeks - it's not in the ambient tanks nor visible in any of the header tanks. First it showed up in just one tank, then two, and today I saw a small amount of it in the third.  We have switched the lines/header tanks between treatments approximately every other week to try to minimize a header tank effect. We've been cleaning/sanitizing tanks once per week over the past month, and bleaching the algae lines twice a week (which injects bleach into the seawater lines and runs through the tubes that flow into the culture tanks).  We definitely have access to peristaltic pumps. There are 6 tanks, so ideally we'd want to sample all 6 (could also sample headers, of which there are 3).  

One concern is that I just fully drained, cleaned and sanitized all tanks this afternoon, so there won't likely be much stringy growth by Saturday.  Also - how time intensive is it?

**Comment by Ellior2** [comment]
Well I am not totally sure yet since we've developed a new filtration method. I anticipate it being ALOT faster. I would guess 30 min for each sample with sanitizing in between and all that. It will also depend on how much algae you have in your water and how much water you want to sample. I did 3.5L last time and since we don't have results on that yet I don't know if that was enough or overkill. Brook suggested 1-10L initially. I should have enough filters leftover to do your 9 samples but will confirm tomorrow morning.

If you are really curious about this stringy stuff, this filtration method is not the best choice, it's just going to clog it. I would just follow Steven's advice and grab some of it to preserve. When I collected my water samples, I was really careful to not stir up any poop or get any big particulate matter as it would just clog instantly.
